<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TescoCamClass</class>
 <widget class="QWidget" name="TescoCamClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TescoCam</string>
  </property>
  <widget class="QLabel" name="imageHolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Image goes here</string>
   </property>
  </widget>
  <widget class="QLabel" name="brightnessLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Brightness</string>
   </property>
  </widget>
  <widget class="QLabel" name="contrastLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Contrast</string>
   </property>
  </widget>
  <widget class="QLabel" name="saturationLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Saturation</string>
   </property>
  </widget>
  <widget class="QLabel" name="whiteBalLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>White Balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="redBalLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Red Balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="blueBalLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue Balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="gainLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain</string>
   </property>
  </widget>
  <widget class="QLabel" name="autoGainLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>150</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Gain</string>
   </property>
  </widget>
  <widget class="QLabel" name="exposureLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>170</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure</string>
   </property>
  </widget>
  <widget class="QLabel" name="brightnessVal">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>10</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="contrastVal">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>30</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="saturationVal">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>50</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="whiteBalVal">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>70</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="redBalVal">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>90</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="blueBalVal">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>110</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="gainVal">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>130</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="autoGainVal">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>150</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="exposureVal">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>170</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSlider" name="brightnessSlider">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="contrastSlider">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="whiteBalSlider">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>70</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="redBalSlider">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>90</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="blueBalSlider">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="gainSlider">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>130</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="autoGainSlider">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="exposureSlider">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>170</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="saturationSlider">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="calibrateButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>190</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="correctedCheckBox">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>195</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Corrected Image</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>641</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="glHorizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
